lib/ansible/modules/cloud/azure/azure_rm_rediscache.py

```diff
@@ -562,8 +576,11 @@ def create_rediscache(self):
             response = self._client.redis.create(resource_group_name=self.resource_group,
                                                  name=self.name,
                                                  parameters=params)
```

Comment:
maybe we shoudl submit a bug to SDK guys, i thought the purpose of Poller was to wait until the resource is completely provisioned

Reply:
actually the line of checking response is AzureOperationPoller or LORPoller is useful in redis, because redis.create directly return a RedisResource.

Reply:
for long running operation, it's always better to give user an option instead of directly poll/wait  the call  to wait for something, given already knowing it's a long run operation.